templates/angular-website-template/app/angular.json

```diff
@@ -56,6 +56,30 @@
                   "maximumError": "5mb"
                 }
               ]
+            },
+            "dev": {
```

Comment:
To keep it consistent, can we use `prod` (similar to `dev`) instead of `production`?

Reply:
ok

Reply:
Done the changes as per the comment.